dashboard/app/models/sections/section.rb

```diff
@@ -155,8 +198,9 @@ def self.valid_participant_type?(type)
     Curriculum::SharedCourseConstants::PARTICIPANT_AUDIENCE.to_h.value?(type)
   end
 
-  def self.valid_grade?(grade)
-    SharedConstants::STUDENT_GRADE_LEVELS.include?(grade) || grade == PL_GRADE_VALUE
+  def self.valid_grades?(grades)
+    return false if grades.empty?
+    (grades - VALID_GRADES).empty?
```

Comment:
there's a weird edge case here where `nil` is a valid value for `grades`, but raises an exception on a `valid_grades?` check. to match the old behavior, i think we'd change this to return `false` if `grades` is `nil`. I wasn't sure if I should do that or not - so let me know if anyone has an opinion here!

Reply:
I think this is fine, but definitely worth monitoring after deploying just in case!